staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1732,6 +1732,11 @@ type VolumeMount struct {
 	// This field is beta in 1.10.
 	// +optional
 	MountPropagation *MountPropagationMode `json:"mountPropagation,omitempty" protobuf:"bytes,5,opt,name=mountPropagation,casttype=MountPropagationMode"`
+	// Expanded path within the volume from which the container's volume should be mounted.
+	// Defaults to "" (volume's root).
+	// SubPathExpr and SubPath are mutually exclusive.
```

Comment:
Perhaps this field should be immediately after SubPath?

Reply:
Please state that this is alpha and in which version (like mount propagation above).

Reply:
In terms of the ordering - I didn't want to interfere with the protobuf numbering / merging, so it is next in the list

Reply:
Added comment